spec/models/section_spec.rb

```diff
@@ -41,4 +41,100 @@
       end
     end
   end
+
+  context 'when sections are created and destroyed' do
+    before do
+      allow(EventSender).to receive(:send_event)
+      @section = create(:section)
+      @section.destroy
+    end
+
+    it 'sends a section_added event' do
+      expect(EventSender).to have_received(:send_event).with(anything, "section_added", any_args)
+    end
+
+    it 'sends a section_removed event' do
+      expect(EventSender).to have_received(:send_event).with(anything, "section_removed", any_args)

```

Comment:
using anything and any_args really removes the value of this test since at this point all its testing is that the callbacks occurred, which are only 1 simple line of code each